README.md

```diff
@@ -160,7 +160,15 @@ As shown in [this great article](http://skarlso.github.io/2016/06/12/google-sign
 
 ![Picture of Google Cloud Console showing API Manager](https://github.com/zalando-techmonkeys/gin-oauth2/blob/feature/google-oauth/doc/gcp-credentials.png?raw=true)
 
-Specify your chosen credentials and scopes for authorization, and redirect the URL to get redirected to after the Google OAuth2 was done, a secret for the CookieStore and a session name to be used in [Sessions as identifier](https://github.com/gin-gonic/contrib/sessions).
+You have to specify a path to your [clientid credential file](./example/test-clientid.google.json) and a slice of
+scopes that you request for authorization.
+You have also to specify the redirect URL to get redirected to, after
+the Google OAuth2 was done.
+As last you have to choose a secret for the CookieStore and a session
```

Comment:
How about?:

You have also to specify the URL to get redirected to upon completion of the Google OAuth2.